modules/spring-config/src/main/java/org/mule/runtime/config/internal/model/DefaultComponentParameterAst.java

```diff
@@ -173,4 +173,14 @@ public ParameterModel getModel() {
     return ofNullable(metadata);
   }
 
+  @Override
+  public boolean isDefault() {
+    return ofNullable(getRawValue()).map(v -> {
+      if (getModel() != null && getModel().getDefaultValue() != null) {
```

Comment:
I'm pretty sure this is also being done when value is set: getModel().getDefaultValue()
Wouldn't be better to set the isDefault=true when the value was set and populated from the getModel().getDefaultValue()? Or maybe I'm wrong and the default value already comes from the XSD.

Reply:
If the default value comes from the XSD we should leave this logic here to be based on the model definition.